SofaKernel/modules/SofaCore/src/sofa/core/MechanicalParams.h

```diff
@@ -277,6 +278,9 @@ class SOFA_CORE_API MechanicalParams : public sofa::core::ExecParams
     /// True if a symmetric matrix is assumed in the left-hand term of the dynamics equations, for solvers specialized on symmetric matrices
     bool m_symmetricMatrix;
 
+    /// True if the call to addForce must store information for a latter call to addDForce
+    bool m_linearizeForces;
```

Comment:
I think it would be useful to have an example of a use of this bool. Is it required to use this variable in all force fields?

Reply:
Ideally yes. Rationale is that you don't want to linealize your forces each time you call addForce (or store linealization point info). But if you don't use it, then nothing bad happens, but you may be losing performance.

Actually, would be more useful to set the proper value of this variable from the solvers, so ForceFields can react to it. To be safe, it is initialize to true, so current behavior is preserved.